Production/test/condorSub/ul_condorSubmit_dictionaries/dict_Summer20UL17_singlet.py

```diff
@@ -0,0 +1,13 @@
+flist = {
+
+   "scenario": "Summer20UL17",
+   "samples": [
+
+       ['Summer20UL17.ST_t-channel_eleDecays_TuneCP5_13TeV-comphep-pythia8'],
```

Comment:
There were a lot more single top file lists added. Either some are missing here or you don't need all of the file lists you added.

Reply:
Actually, I recently realized that some of samples are missing !!!

- for example 2018 qcd condor submit dictionary is empty, 
- or if you compare the each year to each other, the condor submit dictionaries don't have the same amount of samples in it
- And I don't know why, I have to check !